src/ShortScheduleCommand.php

```diff
@@ -50,9 +51,29 @@ public function isRunning(): bool
     }
 
     public function run(): void
+    {
+        $this->pendingShortScheduleCommand->getOnOneServer() ? $this->processOnOneServer() : $this->processCommand() ;
+    }
+
+    private function processOnOneServer()
+    {
+        if (Cache::missing($this->pendingShortScheduleCommand->cacheName())) {
```

Comment:
We can lose a level of indentation here.

Please check whether something is in the cache, and return early.